### PR TITLE
Feature - Update homePage to use ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
image
image
Screenshot (93)

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/pages/homePage.tsx [hunk 1]
```diff
@@ -122,53 +125,61 @@ export const HomePage = (): React.ReactElement => {
...
+                    <Grid.Item size={6} sizeResponsive={{small: 12, medium: 4, large: 3}} key={innerIndex}>
                       <SiteCard site={site} onSiteClicked={onSiteClicked} isEnabled={authManager.getHasJwtPermission(`st-${site.siteId}-vw`)} />
-                    </Grid>
+                    </Grid.Item>
+                  </Grid>
                   ))}
                   {accountSites[account.accountId].length === 0 && (
-                    <Grid item xs={12}>
-                      <Typography color='textSecondary'>
+                    <Grid contentAlignment={Alignment.Start} className={classes.siteCardGrid}>
```
Comment (MoiyadKayda) [inline src/packages/console/src/pages/homePage.tsx]: The grid is inside of a Stack (line 134)

Comment (krishan711) [inline src/packages/console/src/pages/homePage.tsx]: I dont think this needs a grid?

File: src/packages/console/src/pages/homePage.tsx [hunk 2]
```diff
@@ -122,53 +124,57 @@ export const HomePage = (): React.ReactElement => {
...
+                  </Text>
+                  <Text variant='colored'>{`(${account.accountType})`}</Text>
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-ed`) && <Button onClicked={(): void => onManageAccountClicked(account)} text='Manage' />}
+                  <Stack.Item growthFactor={1} shrinkFactor={1} />
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-adm`) && <Button onClicked={(): void => onCreateSiteClicked(account)} text='Create site' />}
+                </Stack>
+                {/* </Box> */}
+                <Spacing variant={PaddingSize.Wide} />
+                {accountSites[account.accountId].map((site: Site, innerIndex: number) => (
+                  <EqualGrid contentAlignment={Alignment.Fill} shouldAddGutters={true}>
```
Comment (krishan711) [inline src/packages/console/src/pages/homePage.tsx]: EqualGrid should be outside the map, there should only be one of these created, not one per site.

File: src/packages/console/src/pages/homePage.tsx [hunk 3]
```diff
@@ -122,53 +124,57 @@ export const HomePage = (): React.ReactElement => {
...
+            <Text tag='p'>
               {'An error occurred. Please try again later.'}
-            </Typography>
+            </Text>
           ) : (
             accounts.map((account: Account, index: number): React.ReactElement => (
-              <Paper key={index} elevation={0} className={classes.accountBox}>
-                <Box width={1} display='flex' justifyContent='start' alignItems='baseline'>
-                  <Typography variant='h5' className={classes.accountName}>
+              <Box variant='bordered'>
```
Comment (krishan711) [inline src/packages/console/src/pages/homePage.tsx]:
```suggestion
              <Box key={index} variant='bordered'>
```

File: src/packages/console/src/pages/homePage.tsx [hunk 4]
```diff
@@ -122,53 +85,50 @@ export const HomePage = (): React.ReactElement => {
...
+                  {accountSites[account.accountId].map((site: Site, innerIndex: number) => (
+                    <SiteCard key={innerIndex} site={site} onSiteClicked={onSiteClicked} isEnabled={authManager.getHasJwtPermission(`st-${site.siteId}-vw`)} />
                   ))}
                   {accountSites[account.accountId].length === 0 && (
-                    <Grid item xs={12}>
-                      <Typography color='textSecondary'>
-                        {'No sites yet. Create one now!'}
-                      </Typography>
-                    </Grid>
+                    <Text variant='header6-note'>
```
Comment (krishan711) [inline src/packages/console/src/pages/homePage.tsx]: should not be a header

Comment (MoiyadKayda) [reply src/packages/console/src/pages/homePage.tsx]: So what variant should I use to display a grey color text with normal font size

Comment (krishan711) [reply src/packages/console/src/pages/homePage.tsx]: `light`, see the other psges i updated

Comment (MoiyadKayda) [reply src/packages/console/src/pages/homePage.tsx]: Done

**Review by krishan711 — APPROVED** [review]

**Pull request merged**